ui/src/reportAnIssue.ts

```diff
@@ -12,10 +12,10 @@ import { getPackageInfo } from "./getPackageInfo";
  * Used to open the browser to the "New Issue" page on GitHub with relevant context pre-filled in the issue body
  */
 export function reportAnIssue(actionId: string, parsedError: IParsedError): void {
-    const { extensionName, extensionVersion } = getPackageInfo();
+    const { extensionName, extensionVersion, bugsUrl } = getPackageInfo();
 
     const body: string = `
-&lt;Please be sure to remove any private information before submitting.&gt;
+&lt;Please fill out the repro steps to help us better understand your issue, and be sure to remove any private information before submitting.&gt;
```

Comment:
I'm assuming  `+&lt` should be `<!--- `. 
Either ways, I don't know which PR ought to be reviewed. :P

Reply:
I'll remove this part from mine.